druid-shell/src/platform/windows/window.rs

```diff
@@ -256,6 +260,24 @@ impl WndState {
             handle.add_idle_callback(move |_| handle2.invalidate());
         }
     }
+
+    fn enter_mouse_capture(&mut self, hwnd: HWND, button: MouseButton) {
+        if self.captured_mouse_buttons == 0 {
+            unsafe {
+                SetCapture(hwnd);
+            }
+        }
+        self.captured_mouse_buttons |= 1 << (button as u32);
+    }
+
+    fn exit_mouse_capture(&mut self, button: MouseButton) {
+        self.captured_mouse_buttons &= !(1 << (button as u32));
+        if self.captured_mouse_buttons == 0 {
+            unsafe {
+                ReleaseCapture();
```

Comment:
`ReleaseCapture` can return `FALSE` in case of error. Let's add some error logging here.

Reply:
Good point, added a `warn!` there :)